LoopKit/Service/Remote/Actions/RemoteAction.swift

```diff
@@ -21,13 +21,13 @@ public enum RemoteAction: CustomStringConvertible, Codable {
     var actionName: String {
         switch self {
         case .carbsEntry:
-            return "Carb Entry"
+            return NSLocalizedString("Carb Entry", comment: "Carb entry action name")
```

Comment:
So this will lookup the string in the *app* bundle, not the LoopKit bundle. `LocalizedString` exists to let you look up strings in LoopKit.  Not an issue if this moves into Loop.  Also, I think I saw that this string is being combined with other (non-localized) strings?

Reply:
I ended up dropping these strings for now in these pull requests but will keep these comments in mind in the future PRs where I may add back.